src/rest.js

```diff
@@ -23,8 +23,8 @@ const CR = '\r'.charCodeAt(0); // always 13, but avoid magic constant
 
 const PODMAN_SYSTEM_ADDRESS = "/run/podman/podman.sock";
 
-/* uid: null for logged in session user; 0 for root; in the future we'll support other users */
-/* Return { path, superuser } */
+/* uid: null for logged in session user, otherwise standard Unix user ID
+ * Return { path, superuser } */
```

Comment:
I am trying to remember why `null` for logged in user is special cased? Is this because of having to wait on `cockpit.user()`? Or the `getAddress` code below?

Wondering now if we can clean this up with `cockpit.info()` in the future. However already there and reviewed by me so ignore :)

Reply:
The little aspect is cockpit.user(), but we can sort this out easily. The big aspect is that it's special in two other regards: it doesn't need superuser for connecting (all other users do), and it looks at `$XDG_RUNTIME_DIR`. If we change `null` to `uid` we could compare `uid == cockpit.info.uid` here instead, that should work. But it's a bit unwieldy with the user promise. But after cockpit.info we can certainly revisit this!

Reply:
Agreed, resolving.